src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -185,7 +185,12 @@
         login admin_user
         post :save, params: {
           user: {
-            login: user.login, realname: 'another real name', email: 'new_valid@email.es', state: 'locked', role_ids: new_global_roles.pluck(:id)
+            login:                user.login,
```

Comment:
Can we also test that a normal user can not update the ``ignore_auth_services`` field?

Reply:
Good idea :+1: Added it now.